README.md

```diff
@@ -44,3 +44,27 @@ module.exports = {
 ```bash
 yarn jest
 ```
+
+
+## Options
+
+This project uses [cosmiconfig](https://github.com/davidtheclark/cosmiconfig), so you can provide config via:
+* a `jest-runner-eslint` property in your `package.json`
+* a `jest-runner-eslint.config.js` JS file
+* a `.jest-runner-eslintrc` JSON file
+
+```js
+module.exports = {
+  "cliOptions": {
+    // ESLint cli options options here
+  },
+  "writeOnFix": // (true|false)
```

Comment:
couldn't this instead be `fix: true` in `cliOptions`?

Reply:
So my thought process was this: The ESLint Node api has a [`fix` option](https://eslint.org/docs/developer-guide/nodejs-api#cliengine) but it works a bit different than `--fix` because it doesn't save to disk the result... In order to do that `outputFixes()` needs to be called.

One options that I thought is: "Always call `outputFixes()` when `fix === true`"... but I was a bit worried about it because then it would break the 1:1 mapping with the ESLint docs... If you think that is a better option I'm happy to change it 😄

Reply:
Also, I agree that the `writeOnFix` feels a awkward so I'm happy to change it

Reply:
Since this is eslint command line options, and on the CLI, `--fix` definitely writes to disk, I think that cliOptions is the right place to put it.

In other words, `cliOptions` to me maps to what `eslint` on the CLI supports - what the eslint API supports won't actually matter to users.

Reply:
Perfect, I agree!